catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -207,21 +207,25 @@ const ExpandedImage: FunctionComponent<Props> = ({
         </ImageWrapper>
       )}
       <InfoWrapper>
-        <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
-          <h2
-            className={`${font('intb', 3)} no-margin`}
-            dangerouslySetInnerHTML={{ __html: displayTitle }}
-          />
-          {displayContributor && (
-            <Space
-              as="h3"
-              v={{ size: 's', properties: ['margin-top'] }}
-              className={font('intb', 5)}
-            >
-              {displayContributor}
-            </Space>
-          )}
-        </Space>
+        {(displayTitle || displayContributor) && (
+          <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
+            {displayTitle && (
+              <h2
+                className={`${font('intb', 3)} no-margin`}
+                dangerouslySetInnerHTML={{ __html: displayTitle }}
```

Comment:
Is there a reason to use `dangerouslySetInnerHTML` here?

Reply:
Because this is the title of a work, which might contain HTML characters (in particular `<i>`)